[instantiate_callbacks] Instantiating callback <lightning.pytorch.callbacks.ModelCheckpoint>
[instantiate_callbacks] Instantiating callback <lightning.pytorch.callbacks.EarlyStopping>
[instantiate_callbacks] Instantiating callback <lightning.pytorch.callbacks.LearningRateMonitor>
Epoch 1: 100%|██████████| 2/2 [00:00<00:00,  9.89it/s, loss=0.342, v_num=y8xq]
/n/home10/pafischer/.local/lib/python3.10/site-packages/lightning/fabric/plugins/environments/slurm.py:166: PossibleUserWarning: The `srun` command is available on your system but is not used. HINT: If your intention is to run Lightning on SLURM, prepend your python command with `srun` like so: srun python my_experiment.py ...
  rank_zero_warn(
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
/n/home10/pafischer/.local/lib/python3.10/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:612: UserWarning: Checkpoint directory checkpoints exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [MIG-c09c7c27-9b0d-56f0-abdb-5d89f9c5b36c]
  | Name  | Type        | Params
--------------------------------------
0 | model | ByteNetArch | 1.0 M
--------------------------------------
1.0 M     Trainable params
0         Non-trainable params
1.0 M     Total params
4.104     Total estimated model params size (MB)
/n/home10/pafischer/.local/lib/python3.10/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:224: PossibleUserWarning: The dataloader, val_dataloader 0, does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` (try 64 which is the number of cpus on this machine) in the `DataLoader` init to improve performance.
  rank_zero_warn(
/n/home10/pafischer/.local/lib/python3.10/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:224: PossibleUserWarning: The dataloader, train_dataloader, does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` (try 64 which is the number of cpus on this machine) in the `DataLoader` init to improve performance.
  rank_zero_warn(
/n/home10/pafischer/.local/lib/python3.10/site-packages/lightning/pytorch/trainer/trainer.py:1600: PossibleUserWarning: The number of training batches (1) is smaller than the logging interval Trainer(log_every_n_steps=50). Set a lower value for log_every_n_steps if you want to see logs for the training epoch.
  rank_zero_warn(
Epoch 0, global step 1: 'val_loss' reached 0.33361 (best 0.33361), saving model to 'checkpoints/epoch=0-step=1-val_loss=0.3336.ckpt' as top 1

Epoch 6:   0%|          | 0/2 [00:00<?, ?it/s, loss=0.23, v_num=y8xq]
Epoch 2, global step 3: 'val_loss' reached 0.17735 (best 0.17735), saving model to 'checkpoints/epoch=2-step=3-val_loss=0.1773-v1.ckpt' as top 3
Epoch 3, global step 4: 'val_loss' reached 0.15684 (best 0.15684), saving model to 'checkpoints/epoch=3-step=4-val_loss=0.1568.ckpt' as top 4
Epoch 4, global step 5: 'val_loss' reached 0.14235 (best 0.14235), saving model to 'checkpoints/epoch=4-step=5-val_loss=0.1423-v4.ckpt' as top 5
Epoch 5, global step 6: 'val_loss' reached 0.13185 (best 0.13185), saving model to 'checkpoints/epoch=5-step=6-val_loss=0.1319-v3.ckpt' as top 6
Epoch 6, global step 7: 'val_loss' reached 0.11716 (best 0.11716), saving model to 'checkpoints/epoch=6-step=7-val_loss=0.1172-v1.ckpt' as top 7
Epoch 7, global step 8: 'val_loss' reached 0.10334 (best 0.10334), saving model to 'checkpoints/epoch=7-step=8-val_loss=0.1033.ckpt' as top 8
Epoch 8, global step 9: 'val_loss' reached 0.09299 (best 0.09299), saving model to 'checkpoints/epoch=8-step=9-val_loss=0.0930-v3.ckpt' as top 9

Epoch 10:   0%|          | 0/2 [00:00<?, ?it/s, loss=0.182, v_num=y8xq]
Epoch 10, global step 11: 'val_loss' reached 0.07775 (best 0.07775), saving model to 'checkpoints/epoch=10-step=11-val_loss=0.0778-v2.ckpt' as top 11
Epoch 11, global step 12: 'val_loss' reached 0.07319 (best 0.07319), saving model to 'checkpoints/epoch=11-step=12-val_loss=0.0732-v2.ckpt' as top 12
Epoch 12, global step 13: 'val_loss' reached 0.06852 (best 0.06852), saving model to 'checkpoints/epoch=12-step=13-val_loss=0.0685.ckpt' as top 13
Epoch 13, global step 14: 'val_loss' reached 0.06623 (best 0.06623), saving model to 'checkpoints/epoch=13-step=14-val_loss=0.0662-v1.ckpt' as top 14

Epoch 14: 100%|██████████| 2/2 [00:00<00:00,  9.76it/s, loss=0.146, v_num=y8xq]
Epoch 15, global step 16: 'val_loss' reached 0.06107 (best 0.06107), saving model to 'checkpoints/epoch=15-step=16-val_loss=0.0611-v6.ckpt' as top 16
Epoch 16, global step 17: 'val_loss' reached 0.06082 (best 0.06082), saving model to 'checkpoints/epoch=16-step=17-val_loss=0.0608-v2.ckpt' as top 17
Epoch 17, global step 18: 'val_loss' reached 0.05902 (best 0.05902), saving model to 'checkpoints/epoch=17-step=18-val_loss=0.0590-v4.ckpt' as top 18
Epoch 19:  50%|█████     | 1/2 [00:00<00:00,  8.52it/s, loss=0.125, v_num=y8xq]

Validation: 0it [00:00, ?it/s]
Epoch 19, global step 20: 'val_loss' reached 0.05724 (best 0.05724), saving model to 'checkpoints/epoch=19-step=20-val_loss=0.0572-v4.ckpt' as top 20
`Trainer.fit` stopped: `max_epochs=20` reached.